### ISSUE TITLE
v9.1.1 fails to install

### ISSUE THREAD

**Issue by mtth** [krzkaczor/ts-essentials#290]
Hi, thanks for this very useful package. The latest version fails to install unfortunately, both with `yarn` and `npm`. With `npm`, it seems it's because [this package now requires `yarn`](https://github.com/krzkaczor/ts-essentials/commit/c338a99a469e0a44c7b64ccbe32d809436a49bce#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R21). And with `yarn` because `scripts/sync-ts-version.js` is not included in the tar file.

```sh
$ yarn add ts-essentials@9.1.1
# ...
Exit code: 1
Command: yarn setTsVersion
$ node scripts/sync-ts-version.js
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '[...]/node_modules/ts-essentials/scripts/sync-ts-version.js'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v17.2.0
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Could you take a look (and allow installing it without `yarn` again)?

**Comment by Beraliv** [comment]
Hey 👋

Thank you for the report and sorry for the inconvenience!

I already know about the problem and will try to fix it asap

I added postinstall script before the last release which causes the problem (it's intended to be used for tests only) - https://github.com/krzkaczor/ts-essentials/pull/283

**Comment by oyed** [comment]
@Beraliv Any updates on this? A package I use depends on `ts-essentials` and is breaking because of this.

**Comment by Beraliv** [comment]
> @Beraliv Any updates on this? A package I use depends on `ts-essentials` and is breaking because of this.

I can quickly add the patch where I check if file exists, if not, I skip this script execution

Let me prepare it in several hours

**Comment by Beraliv** [comment]
I created a fix here – https://github.com/krzkaczor/ts-essentials/pull/291, I checked it locally on my blog project, it should be fine now

**Comment by Beraliv** [comment]
@mtth @oyed I will release it in https://github.com/krzkaczor/ts-essentials/pull/292

Thank you for checking and sorry again for the inconvenience!

**Comment by Beraliv** [comment]
Done ✅

**Comment by chrisnurse** [comment]
Well done to the community and thank you for fixing this so quickly

**Comment by mtth** [comment]
Thanks for the fix @Beraliv - confirming that the issue is fixed in `9.1.2`.
